monkestation/code/modules/uplink/uplink_items.dm

```diff
@@ -30,6 +30,12 @@
 	item = /obj/item/storage/box/syndie_kit/imp_hard_spear
 	cost = 10
 
+/datum/uplink_item/bearserum
```

Comment:
this needs to be typepathed into the matching grouping you want ie race_restricted, implants etc